### ISSUE TITLE
CustomResource showing null values in view

### ISSUE THREAD

**Issue by siju-atlan** [aptakube/aptakube#146]
### Current and expected behavior

While checking to find machines provisioned by Karpenter on an EKS cluster, multiple fields show null value

Current Behaviour:
Type,Zone,Capacity, Provisioner fields show null value
![image](https://github.com/aptakube/aptakube/assets/108319701/b559016e-f167-4bfa-9841-dcab3ac3eb41)

Expected Behaviour:
kubectl get command shows the info correctly
![image](https://github.com/aptakube/aptakube/assets/108319701/a81f5a59-3eff-4ee2-a795-f09b296a1d99)

### Environment & Operating System

macOS 13.4.1
Kubernetes 1.24
Aptakube version: 1.5.3 (20230811.153724)

**Comment by goenning** [comment]
Hi, @siju-atlan thanks for reporting this bug. The root cause is the use of nested keys that have an escaped dot, such as: `.metadata.labels.karpenter\.sh/capacity-type` in their CRD.

I have a fix for this and will release it on the next minor version 1.5.4.

**Comment by siju-atlan** [comment]
Thanks for the quick acknowledgment and for sharing the details of the issue. Appreciate it.
Looking forward to the next release.

**Comment by goenning** [comment]
Hi @siju-atlan this should be fixed on 1.5.4, but please reopen if it's not! Thanks

**Comment by siju-atlan** [comment]
It is partially fixed now. The Capacity and Provisioner sections render correctly. Thanks for the fux

But Type and Zone still show empty fields
<img width="1501" alt="image" src="https://github.com/aptakube/aptakube/assets/108319701/8c1704e0-3bda-4692-9b9b-cc5e432b3209">

**Comment by goenning** [comment]
@siju-atlan what version of the CRD are you using?

**Comment by siju-atlan** [comment]
```
kubectl get crd machines.karpenter.sh -o jsonpath='{.spec.versions[*].name}'
v1alpha5
```

**Comment by goenning** [comment]
Thanks @siju-atlan I just released 1.5.6 which should fix Type and Zone columns too. Can you please have a look?

**Comment by siju-atlan** [comment]
Appreciate the quick and prompt resolution :) 

 I confirm this works fine now.
